### DISCUSSION TITLE
How to solve " 'stdio.h' file not found" message in editor?

### DISCUSSION THREAD

**Discussion by delsindelainexe** [helix-editor/helix#12041] [Troubleshooting] [answered]
# Summary

I decided to try out Helix recently as my code editor for C, I installed, clangd and LLVM, all the things you need, but when I write "<stdio.h>" I'm shown a message in the editor that says " 'stdio.h' file not found", I searched for solutions, generated the compile_commands.json file and still didn't solve my issue and I don't know what to do anymore.

# Enviroment Variables

Just in case these might be useful or in case I'm missing something
![image](https://github.com/user-attachments/assets/c3790ed6-82e4-4cc2-8567-05186a4dd620)

## User Path
![image](https://github.com/user-attachments/assets/a78eb6c9-c450-4dc3-ae74-93b8e23465bd)

## System Path
![image](https://github.com/user-attachments/assets/579544c7-92db-4cc9-b2d6-df2d605efba0)

# Reproduction Steps

All I did was Install Helix, Clangd and LLVM and generate compile_commands.json and such with CMake.

# Helix log

[helix.log](https://github.com/user-attachments/files/17686977/helix.log)

# Platform

Windows 11

# Terminal Emulator

I'm using powershell 7, but the same thing is present in CMD.

# Installation Method

winget install Helix.Helix

# Helix Version

helix 24.7 (079f5442)

**Comment by joao-novo** [comment]
does your program compile or does it show a similar error when you try to do it?

**Reply by delsindelainexe** [reply]
It compiles and runs normally when I run the gcc command in the terminal.

**Reply by joao-novo** [reply]
then it's most likely a problem with your C standard libraries. I'm not sure how it works in windows, so maybe someone else might be able to help you fix them, but try adding them to your path, since it looks like they're not there.

**Reply by delsindelainexe** [reply]
adding what exacly to path? How do I find it?

**Comment by batyrmuhadov** [comment, marked as answer]
I use Clangd and LLVM, and they work flawlessly for me. Here's what I did: I downloaded MinGW from [WinLibs](https://winlibs.com/) extracted it to my preferred folder, added it to my system's environment path, and that’s it.

**Reply by delsindelainexe** [reply]
Dude, I don't know what dark magic this link of yours had but I did as you said and it simply worked, tysm.
